manual/de/ch_GettingStarted.xml

```diff
@@ -226,52 +253,53 @@ Translators:
     </figure>
 
-    <variablelist>
-      <varlistentry>
-        <term><guibutton>Vorheriger</guibutton></term>
+    <para>Mit den Steuerelementen am unteren Rand des Dialogs können Sie die Anzeige dieser Tipps
+      de-/aktivieren und die einzelnen Tipps durchblättern.
+    </para>
+
+    <glosslist>
+      <glossentry>
+        <glossterm>Tipps beim Programmstart anzeigen</glossterm>
+
+        <glossdef>
+          <para>Mit dem Kontrollkästchen können Sie festlegen, ob Sie einen <guilabel>Tipp des
+          Tages</guilabel> bei jedem Programmstart von &app; lesen wollen oder nicht.
+          </para>
+        </glossdef>
+      </glossentry>
+
+      <glossentry>
+        <glossterm>Vorheriger</glossterm>
 
-        <listitem>
+        <glossdef>
           <para>Zeigt den vorherigen Tipp an.
           </para>
-        </listitem>
-      </varlistentry>
+        </glossdef>
+      </glossentry>
 
-      <varlistentry>
-        <term><guibutton>Nächster</guibutton></term>
+      <glossentry>
+        <glossterm>Nächster</glossterm>
 
-        <listitem>
+        <glossdef>
           <para>Geht zum nächsten Tipp weiter.
           </para>
-        </listitem>
-      </varlistentry>
+        </glossdef>
+      </glossentry>
 
-      <varlistentry>
-        <term><guibutton>Schließen</guibutton></term>
+      <glossentry>
+        <glossterm>Schließen</glossterm>
 
-        <listitem>
+        <glossdef>
           <para>Verlässt diese Darstellung mit den hilfreichen <guilabel>Tipps des Tages</guilabel>.
           </para>
-        </listitem>
-      </varlistentry>
-    </variablelist>
-
-    <para>Mit dem Kontrollkästchen <guibutton>Tipps beim Programmstart anzeigen</guibutton> können Sie
-      festlegen, ob Sie einen <guilabel>Tipp des Tages</guilabel> beim jedem Programmstart von &app;
-      lesen wollen oder nicht.
-    </para>
-
-    <para>Auf der Registerkarte <guilabel>Allgemein</guilabel> in den
-      <menuchoice>
-        <guimenu><accel>B</accel>earbeiten</guimenu><guimenuitem><accel>E</accel>instellungen</guimenuitem>
-      </menuchoice>
-      ist möglich, den <guilabel>Tipp des Tages</guilabel> wieder zu aktivieren, falls Sie die
-      Anzeige einmal abgestellt haben. Das <guilabel>Tipp des Tages</guilabel>- Fenster kann auch
-      manuell geöffnet werden, indem Sie im Menü
-      <menuchoice>
-        <guimenu><accel>H</accel>ilfe</guimenu><guimenuitem><accel>T</accel>ipp des
-        Tages</guimenuitem>
-      </menuchoice>
-      auswählen.
+        </glossdef>
+      </glossentry>
+    </glosslist>
+
+    <para>Auf der Registerkarte <xref linkend="prefs-general" /> in den &app;-Einstellungen, die Sie
+      im Menü unter &mc.ed.pref; erreichen, ist möglich, den <guilabel>Tipp des Tages</guilabel> wieder
+      zu aktivieren, falls Sie die Anzeige einmal abgestellt haben. Das <guilabel>Tipp des
+      Tages</guilabel>-Fenster kann auch manuell geöffnet werden, indem Sie im Menü &mc.help.tip; auswählen.
     </para>
```

Comment:
Noch eine unreife Idee:
Da fehlt doch was …
Wenn Sie denken, die Liste der Tips sollte ergänzt werden, so erstellen sie eine https://wiki.gnucash.org/wiki/Simple_Pull_Request mit ihrer Ergänzung in https://github.com/Gnucash/gnucash/blob/maint/doc/tip_of_the_day.list.c